code/ui/Crosshair.cs

```diff
@@ -99,6 +103,27 @@ public Panel SetupCrosshair(Properties crosshairProperties)
 
         return this;
     }
+
+    public override void Tick()
+    {
+        TTTPlayer player = Local.Pawn as TTTPlayer;
+
+        if (player == null)
+        {
+            return;
+        }
+
+        var weapon = player.ActiveChild as Weapon;
+
+        if (weapon == null)
+        {
+            this.Style.Display = DisplayMode.None;
+        }
+        else
+        {
+            this.Style.Display = DisplayMode.Flex;
+        }
+    }
```

Comment:
Would it be better to have a `empty/hands` weapon that ensures no crosshair is created instead of checking each tick?

Here's an example of how `DM98` creates crosshairs for each of their weapons.
https://github.com/Facepunch/dm98/blob/68d346bca7084b3b8cced71d6f75e38803cc9d35/code/weapons/DmWeapon.cs#L240

if you agree, we can create this `empty/hands` weapon in a separate PR also, since it isn't really in the scope of this.

Reply:
Yea this just was planned as a "hotfix", a better implementation is to create / update the Crosshair on WeaponSwitch etc. :)

Reply:
Yup no worries, feel free to leave it as is.